service/frontend/service.go

```diff
@@ -198,6 +198,8 @@ type Config struct {
 	AdminEnableListHistoryTasks dynamicconfig.BoolPropertyFn
 
 	MaskInternalErrorDetails dynamicconfig.BoolPropertyFnWithNamespaceFilter
+
+	EnableEagerWorkflowStart dynamicconfig.TypedPropertyFnWithNamespaceFilter[bool]
```

Comment:
Why not `BoolPropertyFnWithNamespaceFilter`? Are we trying to move all new configs to the generic settings?

Reply:
No good reason, one is a type alias for the other.

Reply:
Changed.